src/v0/destinations/mp/transform.js

```diff
@@ -92,7 +91,10 @@ const responseBuilderSimple = (parameters, message, eventType, destConfig) => {
         const credentials = setImportCredentials(destConfig);
         response.endpoint = credentials.endpoint;
         response.headers = credentials.headers;
-        response.params.project_id = credentials.params?.projectId;
+        response.params = {
+          project_id: credentials.params?.projectId,
```

Comment:
why we are optional chaining here. isn't projectId a required param?
If yes we need to throw error without going forward

Reply:
project is optional parameter.